x/ccv/provider/keeper/proposal_test.go

```diff
@@ -231,6 +231,7 @@ func TestPendingConsumerAdditionPropDeletion(t *testing.T) {
 	defer ctrl.Finish()
 
 	for _, tc := range testCases {
+		tc := tc
```

Comment:
Not sure this is actually an issue?

Can you confirm?

Reply:
not an issue considering how the data is in SetPendingConsumerAdditionProp used but this is a strong assumption and could change. It's safer this way IMO.